opencensus/metrics_quickstart/main.go

```diff
@@ -59,7 +59,12 @@ func main() {
 	// Exporters use Application Default Credentials to authenticate.
 	// See https://developers.google.com/identity/protocols/application-default-credentials
 	// for more details.
-	exporter, err := stackdriver.NewExporter(stackdriver.Options{})
+	exporter, err := stackdriver.NewExporter(stackdriver.Options{
+		// For demonstration the ReportingInterval is set to 10 seconds.
+		// The default value is 60 seconds. Choose the interval appropriately as it
+		// can affect the cost.
```

Comment:
Delete the last sentence? We aren't giving any guidance about how to choose appropriately.

Reply:
I added the last sentence to make sure people are not surprised by the cost as it may be higher if they choose 10 seconds vs 60 seconds.
The only reason I used 10 second is that user doesn't have to wait too long when the are trying it out. If we decided to revert back to 60 seconds then I can remove the comment entirely.

Reply:
I'm OK with a sentence about cost, but saying to choose appropriately without saying what is appropriate is confusing. Is 10 seconds appropriate, given it's in this sample? (I don't think so.)

Reply:
I will remove the 10 second change as per the next comment. So the comment will be removed as well.